javascript-timers/interval.js

```diff
@@ -0,0 +1,12 @@
+var $countdownDisplay = document.querySelector('.countdown-display');
+var text = ['3', '2', '1', '~Earth Beeeelooowww Us~'];
+var counter = 0;
+function countdown() {
+  $countdownDisplay.textContent = text[counter];
+  counter++;
+  if (counter >= text.length) {
+    counter = 3;
+    clearInterval();
+  }
+}
+setInterval(countdown, 1000);
```

Comment:
To capture the id we can do the following: 

`var timerId = setInterval(countdown, 1000);`

You will then pass this to the `clearInterval()` function.